simplq/src/styles/sidePanel.module.scss

```diff
@@ -1,12 +1,17 @@
 @import 'utils';
 
 .side-panel {
-  padding-top: 1.5rem;
-  padding-right: 6rem;
+  box-sizing: border-box;
+  padding: 0 6rem 2.45rem 0;
+  display: flex;
+  flex-flow: column;
 
-  > div {
+  > button {
     padding: 0.5rem 1rem;
-    padding-right: 4rem;
```

Comment:
This sidePanel is used in the admin page too. This change would break that page. Could you take a look at /Admin/AdminSidePanel and convert its components to buttons from divs the same way? Or you could leave '> div' along with '> button' in case you get stuck with modifying the admin sidepanel so that this change doesn't break the admin page.

Reply:
I can take a look. By the way, I just submitted a pull request for a ticket for the AdminPanel, but it covers the Header.

Reply:
Let's convert the admin components to button as well.